### PR TITLE
Update to react 16!

### PR DESCRIPTION
**Pull request by paulkaplan**
Updates all the required packages for react 16!

Fixes the tests that broke when updating to enzyme 3. They had to do with updating component props.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "i18n:msgs": "node ./scripts/generate-locale-messages.js",
     "i18n:src": "babel src > tmp.js && rimraf tmp.js && ./scripts/build-i18n-source.js ./translations/messages/ ./translations/",
     "start": "npm run i18n:msgs && webpack-dev-server",
-    "test": "npm run test:lint && npm run build && npm run test:unit && npm run test:integration",
+    "test": "npm run test:lint && npm run test:unit && NODE_ENV=production npm run build && npm run test:integration",
```
Comment (rschamp) [inline package.json]: We should check if this works on Windows.

File: .travis.yml [hunk 2]
```diff
@@ -11,9 +11,6 @@ node_js:
 cache:
   directories:
   - node_modules
-env:
-  global:
-  - NODE_ENV=production
 install:
 - npm --production=false install
 - npm --production=false update
```
Comment (rschamp) [inline .travis.yml]: We can simplify this and remove `--production=false` if we're not setting `NODE_ENV=production` globally.

File: package.json (hunk 1 above)
Comment (paulkaplan) [reply package.json]: The same change in scratch-paint worked for @fsih on windows

File: .travis.yml (hunk 2 above)
Comment (paulkaplan) [reply .travis.yml]: Fixed in 8ded00f

**Pull request merged**